src/data/storeHelper.ts

```diff
@@ -19,6 +24,44 @@ export function findQueueIndexes(songs: Song[]): number[] {
     return songs.map(({ id }) => q.findIndex((song) => song.id === id));
 }
 
+export function resetDraggedSongs() {
+    if (get(draggedSongs).length) {
+        draggedAlbum.set(null);
```

Comment:
Ok for now with 3 draggables, when we add more a generic type would be preferable.